pkg/controller/storagecluster/noobaa_system_reconciler_test.go

```diff
@@ -266,7 +266,7 @@ func assertNoobaaResource(t *testing.T, reconciler ReconcileStorageCluster) {
 	err = reconciler.client.Get(nil, request.NamespacedName, fNoobaa)
 	assert.NoError(t, err)
 	assert.NotEmpty(t, fNoobaa.Labels[externalRgwEndpointLabelName])
-	assert.Equal(t, fNoobaa.Labels[externalRgwEndpointLabelName], "10.20.30.40:50")
+	assert.Equal(t, fNoobaa.Labels[externalRgwEndpointLabelName], "MTAuMjAuMzAuNDA6NTA=")
```

Comment:
Can you add a comment that this is encoded value for `"10.20.30.40:50"` which we randomly chose? Just to save confusion later on.